electron-app/tests/common/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function capitalize(string: string): string {
+  return string[0].toUpperCase() + string.slice(1);
```

Comment:
is that the same as the filter one? any reason why this is duplicated in the test code?

Reply:
Yes it's the same, I wasn't able to figure out how do I use the filters in the e2e test code. Cypress complains they're not exported if you just do e.g. 
`import { capitalize } from '../../../src/filters';`

Reply:
I was not exporting the filter functions, so you cannot actually import them, but I don't see anything against exporting them in the first place. It would also help us unit-test them too. So instead of duplicating, we can start exporting them.

Reply:
I'm having trouble importing filters because within filters there are import statements that use the `@/whatever` syntax which Cypress errors on.
```
Error: Webpack Compilation Error
./src/filters.ts
Module not found: Error: Can't resolve '@/data/date_formatter' in 'C:\Users\Gebruiker\Source\Repos\rotki\electron-app\src'
resolve '@/data/date_formatter' in 'C:\Users\Gebruiker\Source\Repos\rotki\electron-app\src'
```

How did you manage to import files like this in the unit tests?